### DISCUSSION TITLE
User setup

### DISCUSSION THREAD

**Discussion by MainWire** [Bodmer/TFT_eSPI#2550] [Q&A - General] [answered]
Hi, I am currently starting with the TFT library, and I read everywhere about "`user setup.h`", but I don't know what it is, what to put inside, and where to find that file. I couldn't find any doc, or tutorial on that. Could anyone link me to something ?
How can I modify the default pins of the libary ? Do I have to modifiy them manually or just declare them in my sketch ? _Ex :_ `#define TFT_RST 4` ?
Thks

**Comment by Qyther** [comment, marked as answer]
[Here is a video](https://www.youtube.com/watch?v=rq5yPJbX_uk) detailing how to get it working. Please note that the settings vary on the type of TFT display you are using.
